internal/serverapi/serverapi.go

```diff
@@ -275,4 +275,5 @@ type CLIInfo struct {
 // UIPreferences represents JSON object storing UI preferences.
 type UIPreferences struct {
 	Theme string `json:"theme"` // 'dark', 'light' or ''
+	PageSize string `json:"pageSize"` // 'dark', 'light' or ''
```

Comment:
I recommend updating the comment to show some example values for this field.

Also, I am surprised to see the data type of `string` (as opposed to, for example, `int`). I am not familiar with Go syntax or this `UIPreferences` API, though.